libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/OnOfflineTilesRemovedCallback.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.route.onboard
+
+import com.mapbox.geojson.BoundingBox
+
+/**
+ * Listener that needs to be added to
+ * [MapboxOfflineRouter.removeTiles] to know when the routing
+ * tiles within the provided [BoundingBox] have been removed
+ */
+interface OnOfflineTilesRemovedCallback {
```

Comment:
Same as above re: `internal` modifier

Reply:
This is never used as `RemoveTilesTask` is never used.

Reply:
looks like this callback is for customer

Reply:
As of now, it can only be set on `RemoveTilesTask` which is internal. One or the other needs to be changed.

Reply:
`RemoveTilesTask` cames from legasy codebase and should be fitted to current flow, but it shouldn't be public, but interface `OnOfflineTilesRemovedCallback` should